### ISSUE TITLE
Bad encoding of properties exemple nested in array

### ISSUE THREAD

**Issue by mathroc** [interagent/prmd#234]
When there is utf8 chars in properties exemple they are badly encoded. 

for exemple with this `test.json` file :

```json
{
  "$schema": "http://json-schema.org/draft-04/hyper-schema",
  "definitions": {
    "test": {
      "$schema": "http://json-schema.org/draft-04/hyper-schema",
      "title": "amount",
      "definitions": {
        "amount": {
          "description": "amount",
          "example": "100 €",
          "type": [
            "string"
          ]
        }
      },
      "links": [
        {
          "href": "/test",
          "method": "GET",
          "rel": "self"
        }
      ],
      "properties": {
        "amounts": {
          "type": [
            "array"
          ],
          "items": {
            "$ref": "#/definitions/test/definitions/amount"
          }
        },
        "amount": {
          "$ref": "#/definitions/test/definitions/amount"
        }
      },
      "type": [
        "object"
      ]
    }
  },
  "properties": {
    "test": {
      "$ref": "#/definitions/test"
    }
  },
  "type": [
    "object"
  ]
}
```

when I run `cat test.json | prmd doc`, I have this output:

    ## amount

    ### Attributes
    | Name | Type | Description | Example |
    | ------- | ------- | ------- | ------- |
    | **amounts** | *array* |  | `["100 \u20AC"]` |
    | **amount** | *string* | amount | `"100 €"` |
    ### amount 

    ```
    GET /test
    ```

    #### Curl Example
    ```bash
    $ curl -n -X GET /test

    ```

    #### Response Example
    ```
    HTTP/1.1 200 OK
    ```
    ```json
    {
      "amounts": [
        "100 €"
      ],
      "amount": "100 €"
    }
    ```

as you can see in the `amounts` attribute, "100 €" has been transformed to "100 \u20AC", I think it should not be transformed and behave like in the `amount` attribute or the "Response Example"

**Comment by geemus** [comment]
Hmm. Running that on master, I get:

```
## <a name="resource-test"></a>amount



### Attributes

| Name | Type | Description | Example |
| ------- | ------- | ------- | ------- |
| **amount** | *string* | amount | `"100 €"` |

### amount 



```
GET /test
```


#### Curl Example

```bash
$ curl -n /test
```


#### Response Example

```
HTTP/1.1 200 OK
```

```json
{
  "amounts": [
    "100 €"
  ],
  "amount": "100 €"
}
```
```

Maybe if you update it will match? Also, are you sure the schema you supplied above is the same one that resulted in that output?

**Comment by mathroc** [comment]
I just noticed I was using an old (0.7.0) version of prmd, with prmd 0.7.4 I have the expected output:

    ## amount
    
    
    ### Attributes
    | Name | Type | Description | Example |
    | ------- | ------- | ------- | ------- |
    | **amounts** | *array* |  | `["100 €"]` |
    | **amount** | *string* | amount | `"100 €"` |
    ### amount 
    
    
    ```
    GET /test
    ```
    
    
    #### Curl Example
    ```bash
    $ curl -n -X GET /test
    
    ```
    
    
    #### Response Example
    ```
    HTTP/1.1 200 OK
    ```
    ```json
    {
      "amounts": [
        "100 €"
      ],
      "amount": "100 €"
    }
    ```

and I got that using the json I copied from my post, so I don't know why you don't have the `amounts` attribute in your output. I'll try to build prmd from source to see if I still have the same output

**Comment by geemus** [comment]
Got it. I may well have done something wrong also. Looks like the encoding problem isn't  present any more in the updated output though, which is good. Let me know if you have further issues or questions (or if I just misunderstood). Thanks!

**Comment by mathroc** [comment]
yes, it's solved in 0.7.4 but I have the same problem you had when using a build from master :/ I think the problem first appears with this commit : bce2127bf0c6e0a2205b25f57bcb7cbc6d0be2b5

**Comment by geemus** [comment]
Hey, just tried again (on top of some other fixes) and it seems like this is working as expected again. Sorry for the divergence, hopefully as we continue to build out (and improve test suite) these regressions will become less common. Thanks!
